autonormalize/autonormalize.py

```diff
@@ -92,16 +92,24 @@ def make_entityset(df, dependencies, name=None, time_index=None):
 
     while stack != []:
         current = stack.pop()
+        if (current.df.ww.schema is None):
+            current.df.ww.init(index=current.index[0], name=current.index[0])
```

Comment:
What are the circumstances that require us to reinitialize Woodwork? Are we not retaining the Woodwork typing information when we split the dataframes?

If that's the case, I have a little concern with this behavior. If the user has started with a Woodwork dataframe with certain logical types specified (for example, setting a column as `CountryCode` instead of `Categorical`), these `init` calls will cause Woodwork type inference to re-run which has the potential to cause the types to change.

Is there a way we can retain the typing information so we don't have to reinitialize here? Or if we have to reinitialize can we carry over existing typing information from the original dataframe?

Reply:
Its my understanding that we aren't reinitializing, but instead checking if the df has been initialized at all.